web/client/localConfig.json

```diff
@@ -357,7 +358,7 @@
                   "spatialMethodOptions": [
                       {"id": "Viewport", "name": "queryform.spatialfilter.methods.viewport"},
                       {"id": "BBOX", "name": "queryform.spatialfilter.methods.box"},
-                      {"id": "Circle", "name": "queryform.spatialfilter.methods.circle"},
+                      {"id": "Circle", "name": "queryform.spatialfilter.methods.circle", "geodesic": true},
```

Comment:
please remove these geodesic true from the config
the true should be a default value, not a config option

Reply:
@MV88 maybe this can be left like this due to points in the code like [this](https://github.com/geosolutions-it/MapStore2/blob/9499731916108daa99b6e9dc41ca1524db1c06c1/web/client/components/data/query/SpatialFilter.jsx#L277)

Reply:
i think it can be changed to 
enableGeodesic={selectedMethod.geodesic ? selectedMethod.geodesic : true}

Reply:
i have updated the code above. 

in this way if present in c0onig it will be used, otherwise is true by default

Reply:
@MV88 updated. Thanks for the review.